canonical-web-and-design/maas-ui
# feat(ui): create reusable intro components

## Done

- Create components to capture shared display and behaviour of intro screens.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Create a new user and visit the user intro.
- The intro should appear unchanged.

## Fixes

Fixes: canonical-web-and-design/app-squad#165.
